src/Integration.Vsix.UnitTests/Commands/PackageCommandManagerTests.cs

```diff
@@ -86,10 +87,11 @@ public void PackageCommandManager_Initialize()
             testSubject.Initialize(serviceProvider.GetMefService<ITeamExplorerController>(),
                 serviceProvider.GetMefService<IProjectPropertyManager>(),
                 Mock.Of<IProjectToLanguageMapper>(),
-                Mock.Of<IOutputWindowService>());
+                Mock.Of<IOutputWindowService>(),
+                Mock.Of<IShowInBrowserService>());
 
             // Assert
-            menuService.Commands.Should().HaveCount(allCommands.Count, "Unexpected number of commands");
+            menuService.Commands.Should().HaveCountGreaterOrEqualTo(allCommands.Count, "Unexpected number of commands");
```

Comment:
Same as in the other PR

Reply:
Leyla and I discussed this offline, the problem is with the test expects only the commands in the enum to be registered, whereas now we have commands that are not part of the enum. Hence the `HaveCountGreaterOrEqualTo`. I think we should stop putting code in the PackageCommandManager but for now it's there so this fixes the test. WDYT?